integration-tests/jpa-mariadb/README.md

```diff
@@ -28,3 +28,17 @@ mvn clean install -Dtest-mariadb
 ```
 
 If you have specific requirements, you can define a specific connection URL with `-Dmariadb.url=jdbc:mariadb://...`.
+
+To run the MariaDB server "manually" via command line for testing, the following command line could be useful:
```

Comment:
Maybe it is something I should also add too in my PR https://github.com/quarkusio/quarkus/pull/4067 for mysql-extension.

Reply:
yeah. I used to develop with these around, then today had to figure out the right command again as I hadn't written it down.  Better to take notes :)